command/command_test.go

```diff
@@ -0,0 +1,77 @@
+package command
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/bazelbuild/rules_go/go/tools/bazel"
+)
+
+func TestExecuteProgram(t *testing.T) {
+
+	runfilePath := "command/tests/tests_/tests"
+	testSampleCodePath, err := bazel.Runfile(runfilePath)
+	if err != nil {
+		t.Errorf("Encountered error %s by bazel.Runfile with arg %s", err.Error(), runfilePath)
+	}
+	test := ExternalCommand{
+		Path: testSampleCodePath,
+		Env:  os.Environ(),
+	}
+
+	programstate, err := ExecuteProgram(test, testErrorHandler)
+	if err != nil {
+		t.Errorf("Encountered error %s by ExecuteProgram()", err.Error())
+	}
+
+	want := "Hello\n"
+	got := testWaitOutput(programstate.Stdout, want)
+
+	serr := programstate.Stderr()
```

Comment:
Combine this with line below.

Also, you shouldn't have println or logging statement in tests. If it is something that shouldn't happen, use t.error